### PR TITLE
Entitlement Management  - Auto-assignment doc update

### PR DESCRIPTION
**Pull request by fsheik**

### CODE REVIEW

File: api-reference/v1.0/resources/accesspackageassignmentpolicy.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ To assign a user to an access package, [create an accessPackageAssignmentRequest
 |Property|Type|Description|
 |:---|:---|:---|
 |allowedTargetScope|allowedTargetScope|Principals that can be assigned the access package through this policy. The possible values are: `notSpecified`, `specificDirectoryUsers`, `specificConnectedOrganizationUsers`, `specificDirectoryServicePrincipals`, `allMemberUsers`, `allDirectoryUsers`, `allDirectoryServicePrincipals`, `allConfiguredConnectedOrganizationUsers`, `allExternalUsers`, `unknownFutureValue`.|
+|automaticRequestSettings|[accessPackageAutomaticRequestSettings](../resources/accessPackageAutomaticRequestSettings.md)|The auto assignment settings for the given policy. if this property is set to null, the policy would default to a request based policy.|
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/accesspackageassignmentpolicy.md:30]: Suggest for the second sentence "This property is only present for an auto assignment policy; if absent, this is a request-based policy."

File: api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ms.localizationpriority: medium
+ms.prod: "governance"
+doc_type: resourcePageType
+---
+
+# accessPackageAutomaticRequestSettings resource type
+
+Namespace: microsoft.graph
+
+Specifies if automatic assignments should be enabled, specify grace period before removal and if targets must be removed if they longer match the membership rule specified under `specificAllowedTargets` setting of the access package policy.
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md:14]: Suggest change the end to 

of an [access package assignment policy](accesspackageassignmentpolicy.md).

File: api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|gracePeriodBeforeAccessRemoval|Duration|The number of days access must be retained before the target's access is revoked once they leave the allowed target scope.|
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md:19]: is the grace period supported?  If so, suggest rephrasing as 

The number of days once a target leaves the allowed target scope that their automatic assignment will be retained before their assignment is ended.

File: api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+|removeAccessWhenTargetLeavesAllowedTargets|Boolean|Indicates whether automatic assignment must be removed for targets who move out of the allowed target scope.|
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md:20]: Suggest rephrasing as

If set to true, then when a target leaves the allowed target scope, their automatic assignment will be ended.

File: api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+|requestAccessForAllowedTargets|Boolean|Indicates whether automatic assignments must happen for allowed targets.|
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/accesspackageautomaticrequestsettings.md:21]: Suggest rephrasing as

If set to true, then automatic assignments will be created for targets in the allowed target scope.

File: api-reference/v1.0/resources/attributerulemembers.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+doc_type: resourcePageType
+---
+
+# attributeRuleMembers resource type
+
+Namespace: microsoft.graph
+
+
+
+Used in the specificAllowedTargets setting of an access package assignment policy. The  `@odata.type` value `#microsoft.graph.attributeRuleMembers` identifies a collection of users in the tenant who will be assigned the access package automatically based on the specified membership rule.
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/attributerulemembers.md:16]: As above, let's have "access package assignment policy" be a link to that resource type.

File: api-reference/v1.0/resources/attributerulemembers.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|description|String|A description of the membership rule.|
+|membershipRule|String|The rule that determines members for this access package. For more information about the syntax of the membership rule, see [Membership Rules syntax](/azure/active-directory/enterprise-users/groups-dynamic-membership).|
```
Comment (markwahl-msft) [inline api-reference/v1.0/resources/attributerulemembers.md:23]: Change to "determines the allowed target users for this policy" Iit's not the whole package just one policy)

**Review by markwahl-msft — APPROVED** [review]

File: api-reference/v1.0/resources/accesspackageassignmentpolicy.md (hunk 1 above)
Comment (isvargasmsft) [inline api-reference/v1.0/resources/accesspackageassignmentpolicy.md:30]: In addition, I noticed the beta topics don't have this property. Are you publishing this property in v1 only and not in beta?

File: api-reference/v1.0/resources/attributerulemembers.md (hunk 7 above)
Comment (isvargasmsft) [inline api-reference/v1.0/resources/attributerulemembers.md:23]: Please also list any inherited properties from the base type: subjectSet and subsequent base types of it.

File: api-reference/v1.0/resources/attributerulemembers.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+# attributeRuleMembers resource type
+
+Namespace: microsoft.graph
+
+
+
+Used in the specificAllowedTargets setting of an access package assignment policy. The  `@odata.type` value `#microsoft.graph.attributeRuleMembers` identifies a collection of users in the tenant who will be assigned the access package automatically based on the specified membership rule.
+
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/attributerulemembers.md:17]: Please add a sentence indicating the inheritance: 
```suggestion
Inherits from [subjectSet](subjectset.md).
```

File: api-reference/v1.0/resources/accesspackageassignmentpolicy.md (hunk 1 above)
Comment (isvargasmsft) [reply api-reference/v1.0/resources/accesspackageassignmentpolicy.md:30]: Same comment for the 2 new complex types.

Comment (fsheik) [reply api-reference/v1.0/resources/accesspackageassignmentpolicy.md]: Yes, these properties are v1 only.

File: api-reference/v1.0/resources/attributerulemembers.md (hunk 7 above)
Comment (fsheik) [reply api-reference/v1.0/resources/attributerulemembers.md]: There are no properties in the baseType

**Comment by fsheik** [comment]
> Hello @fsheik,
> 
> Thank you very much for the updates in this PR.
> 
> Please take a look at a few comments for compleness and form.
> 
> In addition, would you be so kind to clarify why we don't have these elements in the beta endpoint?
> 
> Kind regards,
> 
> Isaac

Beta version uses a slightly different API contract, we are retaining that contract until all clients move to v1. Someday we plan to have a deprecation plan for beta and have same contract as that of v1.

File: api-reference/v1.0/api/accesspackageassignmentpolicy-update.md [hunk 9]
```diff
@@ -76,6 +77,11 @@ Content-Type: application/json
   "displayName": "All Users",
   "description": "All users can request for access to the directory.",
   "allowedTargetScope": "allDirectoryUsers",
+  "automaticRequestSettings": {
+      "requestAccessForAllowedTargets": true,
+      "removeAccessWhenTargetLeavesAllowedTargets": true,
+      "gracePeriodBeforeAccessRemoval": null
+  },
```
Comment (isvargasmsft) [inline api-reference/v1.0/api/accesspackageassignmentpolicy-update.md:84]: Would you please add these updates to the Create topic as well?

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**